### PR TITLE
Some updates

### LINKED ISSUE #208: update for new dedicated wizard sections
cf. https://github.com/dobtco/screendoor-v2/pull/2138

### PR DESCRIPTION
**Pull request by aviv**
cf. https://github.com/dobtco/screendoor-v2/pull/2156
closes #208

### CODE REVIEW

**Comment by kariannemah** [comment]
:+1: A few minor inline comments coming up...

File: contents/articles/screendoor/evaluation/advanced_evaluation_settings.md [hunk 1]
```diff
@@ -16,11 +16,13 @@ Once you check this box, a new "Advanced evaluation" section should show up in t
...
+You can decide what percentage of the total rating will come from each rating field. For example, if you want to weigh "Excellence" more heavily than "Originality," you can give it a higher weight. Click "Edit weights" to edit your rating field weights, set them up how you want, and click "Save weights" to save your changes.
+
+> **Note**: The total weight must always add up to 100.
 
 ![field weight](../images/field_weight.png)
 
-If we then rate a response's originality at 5 (out of 10) and its excellence at 10 (out of 10), we will get an average rating of 80% (rather than 75%, as it would be without any weighting). To see details about a weighted rating, simply click the link under "Average rating" on the right-hand side of an individual response page.
+If we then rate a response's originality at 5 (out of 10) and its excellence at 10 (out of 10), we will get an average rating of 88% (rather than 75%, as it would be without any weighting). To see details about a weighted rating, simply click the link under "Average rating" on the right-hand side of an individual response page.
```
Comment (kariannemah) [inline contents/articles/screendoor/evaluation/advanced_evaluation_settings.md]: The first paragraph is in the second person, this ones in the first.

File: contents/articles/screendoor/projects/using_labels.md [hunk 2]
```diff
@@ -10,13 +10,13 @@ Labels (also known as "tags") are another helpful way to organize responses. Res
 
 ## Creating labels
 
-To create a label, click "Add label" on the left-hand side of the "Responses" page. Enter a name for your label (and optionally select a color), then click "Create."
+To create a label, click the "Add label" icon on the left-hand side of the "Responses" page. Enter a name for your label (and optionally select a color), then click "Create."
```
Comment (kariannemah) [inline contents/articles/screendoor/projects/using_labels.md]: I think there needs to be an "and" before "then click..." 

You could instead split it into two sentences.

File: contents/articles/screendoor/response_form/response_preferences.md [hunk 3]
```diff
@@ -39,30 +39,10 @@ To learn more about respondent identification, see the [Identification fields FA
 
 ![data collection](../images/data_collection.png)
 
-### Requiring signatures
-
-If you'd like to require an electronic signature from your respondents, check the the corresponding box. To learn more about signatures, check out the [Signatures](signatures.html) article.
-
 ### Hiding respondent names
 
 To stop your collaborators (other than those with "blind field" permissions) from viewing respondents' names, check the 'Treat respondents names as "blind fields"' box. This will allow you to collect respondents' names while keeping this information from your collaborators/reviewers.
```
Comment (kariannemah) [inline contents/articles/screendoor/response_form/response_preferences.md]: Should the quotation marks be inverted here? i.e., it should read "Treat respondents names as 'blind fields'"

**Comment by ajb** [comment]
Looks :ok_hand: from my end, :+1: to Kari's edits.

File: contents/articles/screendoor/response_form/response_preferences.md (hunk 3 above)
Comment (aviv) [inline contents/articles/screendoor/response_form/response_preferences.md]: I was matching the double quotes in the UI, but yes this is technically more correct.

Comment (kariannemah) [inline contents/articles/screendoor/response_form/response_preferences.md]: Matching the UI, that was my guess! :-)

**Pull request merged**